hardware_interface/include/hardware_interface/actuator_interface.hpp

```diff
@@ -72,6 +72,49 @@ class ActuatorInterface
   virtual
   std::vector<CommandInterface> export_command_interfaces() = 0;
 
+  /// Prepare for a new command interface switch.
+  /**
+   * Prepare for any mode-switching required by the new command interface combination.
+   *
+   * \note this is a non-realtime evaluation of whether a set of command interface claims are
+   * possible, and call to start preparing data structures for the upcoming switch that will occur.
+   * \note switching relevant for other hardware interfaces is also evaluated by this function,
```

Comment:
Can you comment on this comment. It seems to me that this and the following line are saying the opposite. Am I missing something?

Reply:
This was my attempt at commenting on the lack of filtering of the keys from the resource manager. All starting and stopping interfaces requested by the `switch_controller` call are given to all hardware interfaces'  `prepare_command_mode_switch`, but I see that my note is a little confusing, I'll try to reword it :+1: